src/main/java/com/quizapp/enterprise/controllers/GameController.java

```diff
@@ -101,8 +108,9 @@ public void send(WordleDisplayDetail wordleDisplayDetails) throws Exception {
         wordleDisplayDetails.setQuestionId(newQuestion.getQuestionId().intValue());
         wordleDisplayDetails.setTotalGuesses(newQuestion.getTotalGuessesAllowed());
         wordleDisplayDetails.setWordleTimeLimit(newQuestion.getQuestionTimeLimitSeconds());
-        messagingTemplate.convertAndSend("/game1/newQuestion/"  + wordleDisplayDetails.getGameId(), wordleDisplayDetails);
-
+        StringBuilder newQuestionDetailsIdString = new StringBuilder("/game1/newQuestion/")
+                .append(wordleDisplayDetails.getGameId());
+        messagingTemplate.convertAndSend(newQuestionDetailsIdString.toString(), wordleDisplayDetails);
```

Comment:
refactored the string concatenation to use String builder with append.
This change was recommended to by "5. Use StringBuilder or StringBuffer for String Concatenation " section of the java coding best practices summary referenced below.

source: https://xperti.io/blogs/java-coding-best-practices/